packages/core/saved-objects/core-saved-objects-api-server/src/saved_objects_repository.ts

```diff
@@ -348,7 +348,7 @@ export interface ISavedObjectsRepository {
    * It will not create a nested structure like:
    *   `{attributes: {stats: {api: {counter: 1}}}}`
    *
-   * When using incrementCounter for collecting usage data, you need to ensure
+   * When using incrementCounter for collecting, you need to ensure
```

Comment:
```suggestion
   * When using incrementCounter you need to ensure
```
(even) less is more :D